Examples/Physics_applications/plasma_acceleration/README.rst

```diff
@@ -46,17 +66,29 @@ For `MPI-parallel <https://www.mpi-forum.org>`__ runs, prefix these lines with `
          :language: ini
          :caption: You can copy this file from ``Examples/Physics_applications/plasma_acceleration/inputs_3d_boost``.
 
-Analyze
--------
+Visualize
+---------
 
-.. note::
+The plasma has a density profile that consists of a long up-ramp followed by a uniform plateau.
 
-   This section is TODO.
+With this configuration the driver excites a nonlinear plasma wake and drives the bubble depleted of plasma electrons where the beam accelerates, as can be seen in Fig
```

Comment:
as can be seen in \:numref\:\`fig_plasma_acceleration_electron_beam_wakefield_acceleration\`

Reply:
done, thanks!  it looks better